connector/spanmetricsconnector/README.md

```diff
@@ -122,6 +124,8 @@ connectors:
       - name: http.method
         default: GET
       - name: http.status_code
+    exemplars:
+      enabled: false
```

Comment:
`false` is already the default, what do you think of setting this example value to `true`?